repos/spack_repo/builtin/packages/openmpi/package.py

```diff
@@ -737,6 +737,10 @@ def patch(self):
         # https://github.com/open-mpi/ompi/issues/13275#issuecomment-2907903468
         depends_on("prrte")
 
+        for scheduler in [s for s in SCHEDULERS if s not in ("none", "auto", "loadleveler")]:
```

Comment:
I think `none` and `auto` are only added at the variant level, but aren't in `SCHEDULERS` itself.
```suggestion
        for scheduler in [s for s in SCHEDULERS if s not in ("loadleveler")]:
```

Reply:
fixed